### DISCUSSION TITLE
Tile culling

### DISCUSSION THREAD

**Discussion by alienatorZ** [iTowns/itowns#2073] [Q&A]
While working with bathymetry (negative elevations) I am having an issue when I zoom and tilt the camera. The tiles start culling until there is nothing visible.  Below is an example where I am visualizing ocean bathymetry and as I tilt the camera down the horizon starts rapidly dissapearing. I feel that it may have to do with the camera fov but I am not sure. Is there something I can do do avoid this?

Start
![image](https://user-images.githubusercontent.com/5237172/230986680-702e17e7-28f1-48f0-819e-a1fff06908bb.png)

As I tilt/zoom
![image](https://user-images.githubusercontent.com/5237172/230986771-4aa38967-8dc7-4def-b06e-b056cbe5e89f.png)

**Comment by jailln** [comment]
Hi,
May be related with #1981 and #1871 ? Can you try with #1993 to see if it fixes it?

**Reply by alienatorZ** [reply]
How would I do that? I tried to clone the branch and build itowns.js. Then I imported it like `import * as itowns from "./itowns"` But it cant be used. I get itowns.Coordinates cannot be found and a bunch of other errors.

**Reply by mgermerie** [reply]
#1993 has now been merged. You can try to install the `@next` version of iTowns with `npm install --save itowns@next` in your project. This version is up to date with `master` branch.

PS : to resume to iTowns "production" version after testing, you'll just need to run `npm install --save itowns@latest`
